docs: docs for OpenTelemetry instrumentation (#5291)
jina-ai/jina: README.md

docker-compose up
```

> Tracing and monitoring with OpenTelemetry becomes easy:

Reviewer: ```suggestion
Tracing and monitoring with OpenTelemetry is straightforward:
```

class MyExec(Executor):
    @requests
    def encode(self, docs: DocumentArray, **kwargs):
        with self.tracer.start_as_current_span(
            'encode', context=tracing_context
        ) as span:
            with self.monitor(
                'preprocessing_seconds', 'Time preprocessing the requests'
            ):
                docs.tensors = preprocessing(docs)
            with self.monitor(
                'model_inference_seconds', 'Time doing inference the requests'
            ):
                docs.embedding = model_inference(docs.tensors)
```

> Using Grafana becomes easy, just [download this JSON](https://github.com/jina-ai/example-grafana-prometheus/blob/main/grafana-dashboards/flow.json) and import it into Grafana:

Reviewer 2: Edit here, to put something on how u use `Jaegger`?

class MyExec(Executor):
    @requests
    def encode(self, docs: DocumentArray, **kwargs):
        with self.tracer.start_as_current_span(
            'encode', context=tracing_context
        ) as span:
            with self.monitor(
                'preprocessing_seconds', 'Time preprocessing the requests'
            ):
                docs.tensors = preprocessing(docs)
            with self.monitor(
                'model_inference_seconds', 'Time doing inference the requests'
            ):
                docs.embedding = model_inference(docs.tensors)
```

You can integrate Jaeger or any other distributed tracing tools to collect and visualize request-level and application level service operation attributes. This helps you analyze request-response lifecycle, application behavior and performance.

> Using Grafana becomes easy, just [download this JSON](https://github.com/jina-ai/example-grafana-prometheus/blob/main/grafana-dashboards/flow-histogram-metrics.json) and import it into Grafana:

Reviewer: ```suggestion
To use Grafana, [download this JSON](https://github.com/jina-ai/example-grafana-prometheus/blob/main/grafana-dashboards/flow-histogram-metrics.json) and import it into Grafana:
```

<p align="center">
<a href="https://docs.jina.ai"><img src=".github/readme/grafana-histogram-metrics.png?raw=true" alt="Jina: Seamless Container Integration" width="70%"></a>
</p>

> Using Jaeger to trace requests becomes easy.

Reviewer: ```suggestion
To trace requests with Jaeger:
```

                docs.embedding = model_inference(docs.tensors)
```

> Jaeger or any other distributed tracing tool can be integrated to collect and visualize request-level and application level service operation attributes to analyze the request-response lifecycle, application behavior and performance.

Reviewer: ```suggestion
You can integrate Jaeger or other distributed tracing tools to collect and visualize request-level and application level service operation attributes. This helps you analyze request-response lifecycle, application behavior and performance.
```

- Stick to active tense. (i.e. "you can integrate foo" not "foo can be integrated") so it's clear WHO can do what
- Use short sentences
